src/sonic-config-engine/translate_acl

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python
+
+import openconfig_acl 
+import pyangbind.lib.pybindJSON as pybindJSON
+
+import sys
+import json
+
+def dump_json(filename, data):
+    with open(filename, 'w') as outfile:
+        json.dump(data, outfile, indent=4, sort_keys=True, separators=(',', ':'))
+
+def generate_rule_json(table_name, rule):
+    rule_idx = rule.config.sequence_id
+    rule_props = {}
+    rule_data = {}
+    rule_data["ACL_RULE_TABLE:"+table_name+":Rule_"+str(rule_idx)] = rule_props
+    rule_data["OP"] = "SET"
+    
+    rule_props["priority"] = 10000 - rule_idx
```

Comment:
make 10000 as a parameter with default value, it should not be hardcoded in the code.